### PR TITLE
Added perftool-testsuite test

### PR DESCRIPTION
**Pull request by PraveenPenguin**
Added test which run perftool-testsuite and notify
user when test fail

Signed-off-by: Praveen K Pandey <praveen@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
job log : https://paste.fedoraproject.org/458457/20522314/

File: perf/perftool.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        for package in ['gcc', 'make', 'perf']:
+            if not smm.check_installed(package) and not smm.install(package):
+                self.error('%s is needed for the test to be run' % package)
+
+        locations = ["https://github.com/rfmvh/perftool-testsuite/archive/"
+                     "master.zip"]
+        tarball = self.fetch_asset("perftool.zip", locations=locations,
+                                   expire='7d')
+        archive.extract(tarball, self.srcdir)
+        self.srcdir = os.path.join(self.srcdir, 'perftool-testsuite-master')
```
Comment (narasimhan-v) [inline perf/perftool.py]: Are you sure you can change `self.srcdir` and not have any effects in any test ? Please clarify

File: perf/perftool.py [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+    def test(self):
+
+        for line in build.run_make(self.srcdir, extra_args='check',
+                                   ignore_status=True).stdout.splitlines():
+            if 'FAIL' in line:
+                self.fail("Test failed")
```
Comment (PraveenPenguin) [inline perf/perftool.py]: not very sure about this i think log will have all the failure as user can any why need to go through the log , what you think ?

File: perf/perftool.py (hunk 1 above)
Comment (narasimhan-v) [inline perf/perftool.py]: @PraveenPenguin I agree with this, but as a general practice, is it right ? I see many tests doing this recently..

Comment (PraveenPenguin) [inline perf/perftool.py]: @narasimhan-v i think so should be ok

File: perf/perftool.py (hunk 2 above)
Comment (PraveenPenguin) [inline perf/perftool.py]: sure will do

**Comment by PraveenPenguin** [comment]
raised PR #130 with suggested change

**Pull request closed without merging**